src/screens/market/MarketScreen/index.tsx

```diff
@@ -1,14 +1,174 @@
+      value: toFormattedNumber(csprMarketInfo.circulating_supply),
+    },
+  ];
+
+  useEffect(() => {
+    getPriceHistoryInfo();
+  }, []);
+
+  const getPriceHistoryInfo = () => {
+    dispatch(
+      allActions.market.getPriceHistory((error: any) => {
+        setRefreshing(false);
+        if (error) {
+          const message = {
+            message: error && error.message ? error.message : 'Error',
+            type: MessageType.error,
+          };
+          dispatch(allActions.main.showMessage(message));
+        }
+      }),
+    );
+  };
+
+  const onRefresh = () => {
+    setRefreshing(true);
+    getPriceHistoryInfo();
+  };
+
+  const onActivated = () => {
+    if (isScrollable) {
+      setScrollable(false);
+    }
+  };
+
+  const onDeactivated = () => {
+    if (!isScrollable) {
+      setScrollable(true);
+    }
+  };
+
+  const _renderRowInfo = () => {
+    return (
+      <Row style={styles.rowInfoContainer}>
+        {rowInfo.map((item, index) => {
+          return (
+            <Col key={index} style={styles.infoItem}>
+              <Text style={styles.label}>{item.title}</Text>
+              <Text style={styles.value}>{item.value}</Text>
+            </Col>
+          );
+        })}
+      </Row>
+    );
+  };
 
   return (
-    <View style={styles.container}>
-      <Text>MarketScreen</Text>
-    </View>
+    <CLayout
+      edges={['top', 'left', 'right']}
+      bgColor={colors.cF8F8F8}
+      statusBgColor={colors.cF8F8F8}>
+      <View style={styles.container}>
+        <Row.L
+          pl={24}
+          pr={16}
+          pt={10}
+          pb={16}
+          style={{ backgroundColor: colors.cF8F8F8 }}>
+          <Row style={styles.alignCenter}>
+            <IconLogo width={scale(28)} height={scale(28)} />
+            <Text style={[textStyles.H3, { marginLeft: scale(16) }]}>
+              Market
+            </Text>
+          </Row>
+        </Row.L>
+        <ScrollView
+          ref={scrollViewRef}
+          scrollEnabled={isScrollable}
+          refreshControl={
+            <RefreshControl
+              refreshing={isRefreshing}
+              onRefresh={onRefresh}
+              style={{ backgroundColor: colors.cF8F8F8 }}
+            />
+          }
+          showsVerticalScrollIndicator={false}>
+          <Col px={16} style={{ backgroundColor: colors.cF8F8F8 }}>
+            <Text style={styles.head1}>CASPER (CSPR)</Text>
+            <Row mt={8} ml={8} style={styles.alignCenter}>
+              <Text style={textStyles.H5}>
+                {toFormattedCurrency(currentPrice, {
+                  maximumSignificantDigits: 4,
+                })}
+              </Text>
+              {csprMarketInfo.price_change_percentage_24h < 0
+                ? getIcon('down')
+                : getIcon('up')}
+              <Text style={styles.label}>
+                {toFormattedNumber(csprMarketInfo.price_change_percentage_24h, {
+                  maximumSignificantDigits: 3,
```

Comment:
pls help to update the color of the text, if < 0 red, >=0 green

Reply:
Updated